### ISSUE TITLE
Regarding Various input devices (Keyboard, Mouse, Touch, Motion(Comming Soon)…)

### ISSUE THREAD

**Issue by vilav** [naver/egjs-view360#34]
Hi there

Looking fwd to the support for input devices, great to see they're on the roadmap.
Have done some research to find a performant solution and this is very well done indeed!

As an accessibility feature for users who don't immediately grasp the ability to manipulate the image sequence, would be nice to have an HTML range slider that can then be dragged to have the 360 move in sync.

Thanks

**Comment by jongmoon** [comment]
@vilav Thank you for your interest! 👍 

First, related with input devices. spin viewer currently supports touch and mouse only but we will support others(motion, keyboard) as soon as possible. 

I will review the function of the Range slider you mentioned.

Thank you again!

**Comment by vilav** [comment]
@jongmoon did you have any thoughts about that Range slider as as controller for the rotation?

**Comment by jongmoon** [comment]
@vilav So sorry for late response. I've discussed this issue with my co-worker. And we agree Range slider provide convenience and accessibility. But we decided not applying as default functionality.

Because management difficulty of UI/CSS/Looks, So if we provide this, we must consider design it look and how it layouts to provide more functionality. Unfortunately we lack a sense of design.

So we think It's better to make Interface to make it easily.

Currently we provide `setBy` method to rotate. But It seems to be difficult to interlock with range slider because it is relative rotation. **But If we provide `setTo` API which can be used specifying absolute angle.** It's expected to be easy to apply. 

For example, 

```js

slider.oninput = function() {//slider event handler
  spinViewer.setTo({angle: this.value}); //update to that value.
}
```

So we decided to make that API and example that works with range slider. @vilav how do you think about it?

**Comment by jongmoon** [comment]
@vilav Thank you! I am curious about your research results.👍

**Comment by vilav** [comment]
Hi @jongmoon, need to build a prototype for this slider that can move the 360. is there a temporary code / demo you have where a slider is usable with the 360?

**Comment by jongmoon** [comment]
@vilav Sorry for late! I was not able to get your request properly due to other service development work. Until tomorrow, I will reflect the code and demo in my master branch. And I want you to review our codes.

Thank you very much for your long wait.

**Comment by vilav** [comment]
Thank you @jongmoon, looking forward to it!

**Comment by jongmoon** [comment]
@vilav I just PR for this issue. #140 

As I mentioned above, I added setTo method. (named `spinTo`)

And I added range slider to control spinning on manual test code(/test/manual/SpinViewer/SpinViewr.html)

![rangeslider-spinviewer](https://user-images.githubusercontent.com/3255471/35727917-2b6360cc-084d-11e8-8d31-c235594704bb.gif)

Is this an implementation that fits your intentions? If not, please provide feedback and we will reflect it as much as possible.

After reviewing code, I'll merge it.

**Comment by jongmoon** [comment]
Resolved by #140
